README.md

```diff
@@ -31,6 +31,32 @@ You will get:
 
 It is recommended to develop against the `dev` version, and in production to use the `min` version.
 
+## Concepts
+
+Monaco editor is best known for being the text editor that powers VS Code. However, it’s a bit more nuanced. Some basic understanding about the underlying concepts is needed to use Monaco editor effectively.
+
+### Models
+
+Models are at the heart of Monaco editor. It’s what you interact with when managing content. A model represents a file that has been opened. This could represent a file that exists on a file system, but it doesn’t have to. For example, the model holds the text content, determines the language of the content, and tracks the edit history of the content.
+
+### URIs
+
+Each model is identified by a URI. This is why it’s not possible for two models to have the same URI. Ideally when you represent content in Monaco editor, you should think of a virtual file system that matches the files your users are editing. For example, you could use `file:///` as a base path. If a model is created without a URI, it’s URI will be `inmemory://model/1`. The number increases as more models are created.
```

Comment:
```suggestion
Each model is identified by a URI. This is why it’s not possible for two models to have the same URI. Ideally when you represent content in Monaco editor, you should think of a virtual file system that matches the files your users are editing. For example, you could use `file:///` as a base path. If a model is created without a URI, its URI will be `inmemory://model/1`. The number increases as more models are created.
```

In general I think we use `'` instead of `’`. I'd keep that for consistency reasons.